src/utils/utils.ts

```diff
@@ -1 +1,74 @@
-// Write utilities here
+import { Announcement } from "@/models/Announcement";
+
+/**
+ * Filters out announcements whose startDate is in the past.
+ *
+ * @param announcements - An array of Announcement objects.
+ * @returns An array of Announcement objects where each startDate is greater than the current date.
+ */
+export const filterFutureAnnouncements = (
```

Comment:
As mentioned in a comment below, add an optional parameter with a default value (current time) to filter based on the passed in date.